### PR TITLE
[ZH] Translation 12-1

### PR DESCRIPTION
**Pull request by xingdi-eric-yuan**
A few things either I'm not sure or I think the English version is problematic:

In docs/zh/week12/12-1.md:

1. L124: I'm not sure what does "Revolutionized penalty" mean here.
2. L213: Regarding the mask. It should be a `lower triangular matrix`, where bottom left are `1`s, and upper right are `0`s. In this case, say the 3rd row of the mask, only the 1st/2nd/3rd columns have non-zero values, which means the model can only observe these inputs. However, in the original English version, it says:
```
The mask is an upper triangular matrix that have zeros in the lower triangle and negative infinity in the upper triangle. 
```
Which if I understood correctly, is wrong.

### CODE REVIEW

File: docs/zh/week12/12-1.md [hunk 1]
```diff
@@ -0,0 +1,390 @@
+---
+lang-ref: ch.12-1
+title: Deep Learning for NLP
```
Comment (Atcold) [inline docs/zh/week12/12-1.md:3]: Translation?

File: docs/zh/week12/12-1.md [hunk 2]
```diff
@@ -0,0 +1,390 @@
+---
+lang-ref: ch.12-1
```
Comment (Atcold) [inline docs/zh/week12/12-1.md:2]: `lang: zh` is missing.

File: docs/zh/week12/12-1.md [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+title: Deep Learning for NLP
+lecturer: Matt Lewis
+authors: Jiayu Qiu, Yuhong Zhu, Lyuang Fu, Ian Leefmans
+date: 20 Apr 2020
+translator: Eric Yuan
+translation-date: 27 June 2020
```
Comment (Atcold) [inline docs/zh/week12/12-1.md:8]: `Jun`, without the `e`.

File: docs/zh/week12/12-1.md (hunk 1 above)
Comment (xingdi-eric-yuan) [reply docs/zh/week12/12-1.md]: Done

File: docs/zh/week12/12-1.md (hunk 2 above)
Comment (xingdi-eric-yuan) [reply docs/zh/week12/12-1.md:3]: Done

File: docs/zh/week12/12-1.md (hunk 3 above)
Comment (xingdi-eric-yuan) [reply docs/zh/week12/12-1.md]: Done

**Comment by xingdi-eric-yuan** [comment]
Fixed website layouts and added missing labels. Please let me know about the 2 questions at the top, then I'll update them in the translation.

**Comment by Atcold** [comment]
@ileefmans, can you help with Q1? I have no idea what it means.
About Q2, 0 and −∞ are sent to an exponential, which gives you 1 and 0 respectively.

**Comment by xingdi-eric-yuan** [comment]
I see, then for Q2, I would note in the translation that an exp will be applied to that mask.

**Comment by Atcold** [comment]
More than an exp I think it goes through a softmax.

**Comment by Atcold** [comment]
Jeez… So, this was a dump from the auto captioning.
What the speaker says is "it has revolutionised NLP", where NLP stands for Natural Language Processing.

This can be considered solved, @xingdi-eric-yuan.

**Pull request merged**